docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Interactive3D.htm

```diff
@@ -0,0 +1,162 @@
+      height=&quot;240&quot;
+      position=&quot;0 0 0&quot;
+      scale=&quot;50 50 50&quot;
+      rotation=&quot;0 -90 0&quot;
+      light=&quot;0xbbbbbb 0x000000 2&quot;
+      controls
+      autoplay
+      src=&quot;assets/Duck.gltf&quot;
+    &lt;/div&gt;
+</pre>
+
+</li>
+
+</ul>
+
+<h2><a id="options-for-widget-interactive3D-mobile"></a>Options</h2>
+
+	<h3>Summary</h3>
+
+	<table>
+		<tbody>
+		<tr>
+			<th>Option</th>
+			<th>Input type</th>
+			<th>Default value</th>
+			<th>Description</th>
+		</tr>
+
+
+			<tr>
+				<td class="option"><span style="font-family: Courier New,Courier,monospace">src</span></td>
+				<td class="option">string</td>
+				<td class="option">""</td>
+				<td class="option">A string containing the path/URL of the file to be loaded.</td>
+			</tr>
+
+			<tr>
+				<td class="option"><span style="font-family: Courier New,Courier,monospace">width</span></td>
+				<td class="option">string</td>
+				<td class="option">"400"</td>
+				<td class="option">Set the interactive 3D width in pixels.</td>
+			</tr>
+
+			<tr>
+				<td class="option"><span style="font-family: Courier New,Courier,monospace">height</span></td>
+				<td class="option">string</td>
+				<td class="option">"300"</td>
+				<td class="option">Set the interactive 3D height in pixels.</td>
+			</tr>
+
+			<tr>
+				<td class="option"><span style="font-family: Courier New,Courier,monospace">position</span></td>
+				<td class="option">string</td>
+				<td class="option">"0 0 0"</td>
+				<td class="option">Set the interactive 3D local position(x, y, z).</td>
+			</tr>
+
+			<tr>
+				<td class="option"><span style="font-family: Courier New,Courier,monospace">scale</span></td>
+				<td class="option">string</td>
+				<td class="option">"1 1 1"</td>
+				<td class="option">Set the interactive 3D local scale(x, y, z).</td>
+			</tr>
+
+			<tr>
+				<td class="option"><span style="font-family: Courier New,Courier,monospace">rotation</span></td>
+				<td class="option">string</td>
+				<td class="option">"0 0 0"</td>
+				<td class="option">Set the interactive 3D local rotation in radians(x, y, z).</td>
+			</tr>
+
+			<tr>
+				<td class="option"><span style="font-family: Courier New,Courier,monospace">light</span></td>
+				<td class="option">string</td>
+				<td class="option">"0xffffff 0xffffff 1"</td>
+				<td class="option">A light source positioned directly above the scene(skyColor, groundColor, intensity).</td>
+			</tr>
+
+			<tr>
+				<td class="option"><span style="font-family: Courier New,Courier,monospace">controls</span></td>
+				<td class="option"></td>
+				<td class="option"></td>
+				<td class="option">Obrit controls allow the camera to orbit around a target.</td>
+			</tr>
+
+			<tr>
+				<td class="option"><span style="font-family: Courier New,Courier,monospace">autoplay</span></td>
+				<td class="option"></td>
+				<td class="option"></td>
+				<td class="option">Play animations of the model automatically.</td>
```

Comment:
Is there no default value of "autoplay"?
The "autoplay" has the "true, false" value ? or has the another value?

Reply:
Yes. There is no fault value. The execution of the attribute is determined by checking whether it is defined or not.